public/js/nfts.js

```diff
@@ -56,11 +57,34 @@ function renderNFTs(query = '') {
   domTotalHolders.innerText = arrOwners.length.toLocaleString('en-GB');
 
   // Total Burns
-  domTotalBurns.innerText = nTotalBurns.toLocaleString('en-GB');
+  if (nTotalBurns > 0){
+  	domTotalBurns.innerText = nTotalBurns.toLocaleString('en-GB');
+  }
+  else{
+	getInfoCollection();
+  }
 }
 
 // Fetch and render the collection!
 $.getJSON('https://stakecubecoin.net/web3/scp/tokens/getcollection/' + id, function(res) {
     data = res;
     renderNFTs();
-});
\ No newline at end of file
+});
+
+function getInfoCollection(){
+	$.getJSON('https://stakecube.io/api/v2/marketplace/collection?contract=' + id, function(res) {
```

Comment:
Tbh, im not sure about this one in general. I would vote **against** pulling stats from a _centralized third party_. This makes SCPscan depending on others (even its from the same Devs) and one advantage is, SCPscan can run standalone

cc @JSKitty

Reply:
We can find other values to display if there are no NFTs burned in the collection. However, if the marketplace API were no longer functional, those values would not be entered.

Reply:
We can find other values to display if there are no NFTs burned in the collection. However, if the marketplace API were no longer functional, those values would not be entered.

Reply:
I think it's fine to display some centralized info if it is non-mandatory data, like the Verification data is pulled for the Collections ranking, but if the request fails, the site will just fallback to other data, it won't break, a similar method here would be awesome.